### ISSUE TITLE
Clarify Copyright Assignment for Future License Upgrades

### ISSUE THREAD

**Issue by zacharyDez** [developmentseed/cloud-optimized-geospatial-formats-guide#52]
There are discussions about where the guide will be hosted, and we should revisit attribution and licensing pending that conversation. 

**Why This Is Important** 
Understanding who holds the copyright will help us:

- Make informed decisions about license changes in the future.
- Ensure that we have the legal flexibility to adapt as circumstances require.
- Clarify the rights and responsibilities of contributors.

**Questions to Resolve**

- Are contributors implicitly assigning copyright when they contribute, and if so, to whom?
- Do we need to update our CONTRIBUTING.md or other documentation to make this explicit?

**Next Steps**
-  Discuss and share decision within this issue.
 - Update documentation as required.

**Comment by zacharyDez** [comment]
@wildintellect do we consider this solved?

**Comment by wildintellect** [comment]
@zacharyDez no, https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/blob/main/contributing.qmd does not explain authorship credit or copyright ownership.

**Comment by wildintellect** [comment]
@christine-e-smit I think you misunderstand CC, CNGF owns the copyright and is licensing it as CC Attribution there is no conflict. So you are allowed to use it as long as you cite CNGF. I'll ask them to clarify/update to Forum cc: @PowerChell

**Comment by christine-e-smit** [comment]
Ah! Okay. That makes sense. I thought you could only have one or the other. Thank you for the explanation.

**Comment by maxrjones** [comment]
FWIW I think the LICENSE in the repo and the reference to that license in the contributing guide is completely sufficient for people to understand the copyright and licensing situation and leaving this issue open probably just adds confusion.

**Comment by wildintellect** [comment]
@maxrjones I think we need another line in the contribution guide that explicitly states the answer to:
Are contributors implicitly assigning copyright when they contribute, and if so, to whom?

I don't think we ever decided, as the Citation and Copyright notice differ.
